homeassistant/components/smartthings/climate.py

```diff
@@ -499,12 +500,20 @@ def temperature_unit(self) -> str:
 
     def _determine_swing_modes(self) -> list[str]:
         """Return the list of available swing modes."""
+        supported_swings = [SWING_OFF]
```

Comment:
The default should be `None` as if there are no swing modes it should not set this capability at all

Reply:
You right, fixed